### PR TITLE
Adding more facts to dellos9_facts module

### PR DESCRIPTION
**Pull request by maugli13**
This change will give an option to check stacked configurations for DellOS9 switches as well as gathering serial numbers from all systems configured in stack

##### SUMMARY
This change required to gather more detailed information from Force10 switches aggregated in  stack

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME
dellos9_facts

##### ANSIBLE VERSION

```
ansible 2.5.1
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/murmanov/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.12 (default, Dec  4 2017, 14:50:18) [GCC 5.4.0 20160609]

```

##### ADDITIONAL INFORMATION

Basically I've added new facts which can be helpful sometimes

```
        "ansible_net_stack": "Daisy chain", 
        "ansible_net_stack_serialnums": [
            "CN2829851B0244", 
            "CN2829851B0148"
        ], 
        "ansible_net_system_members": [
            "  0   Management   online         MXL-10/40GbE", 
            "  1   Standby      online         MXL-10/40GbE"
        ], 

```

### CODE REVIEW

**Comment by maugli13** [comment]
Tests in Shippable should be adjusted as well, since they don't have correct example from stacked devices to parse

**Comment by mattclay** [comment]
CI failure in unit tests: https://app.shippable.com/github/ansible/ansible/runs/62228/3/tests

**Comment by gundalow** [comment]
Getting close to 2.6 cut off, are you working on fixing this?

**Comment by mattclay** [comment]
In addition to the sanity test failures the unit tests are failing:

https://app.shippable.com/github/ansible/ansible/runs/64921/3/tests

**Comment by skg-net** [comment]
@maugli13 any update?

**Comment by maugli13** [comment]
@skg-net Sorry for delay with answer, at this moment I don't have access to Force10 switches, so not sure if I can work further on this

**Comment by Akasurde** [comment]
@maugli13 is no longer have access to Force10 switches, so closing this PR. Please feel free to open a new PR with latest code. Thanks.
